app/user/user_service.py

```diff
@@ -0,0 +1,47 @@
+from .user_model import User, UserRole
+from app import db
+from flask_jwt_extended import current_user
+from typing import Dict
+
+class UserService:
+    def create_user(self, firstName, lastName, email, password):
+        user = User(firstName=firstName, lastName=lastName, email=email)
+        user.set_password(password=password)
+        user.role = UserRole.STANDARD.value #do I need .value here?
```

Comment:
Yes you need `.value` here. `.value` is assigning the associated value that `UserRole.Standard`. If you don't use `.value` then you are setting the enum case itself as the variable. In this case `user.role` is expecting a string for the DB, without `.value` we will be assigning a class and the DB will throw an error.

Reply:
Let's remove this comment before merging as well